utils/metrics/metrics/src/main/java/com/ea/orbit/metrics/MetricsManager.java

```diff
@@ -256,7 +256,7 @@ private Metric buildGaugeForField(Field field, Object obj)
 
         if (obj != null)
         {
-            for (Field field : obj.getClass().getDeclaredFields())
+            for (Field field : obj.getClass().getFields())
```

Comment:
Thats ok for now. We didn't register inaccessible fields anyway. I plan on looking at this for a future change.